common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/SampleRequestBuilder.java

```diff
@@ -0,0 +1,47 @@
+package ai.h2o.mojos.deploy.common.transform;
+
+import ai.h2o.mojos.deploy.common.rest.model.Row;
+import ai.h2o.mojos.deploy.common.rest.model.ScoreRequest;
+import ai.h2o.mojos.runtime.frame.MojoColumn;
+import ai.h2o.mojos.runtime.frame.MojoFrameMeta;
+
+import static java.util.Arrays.asList;
+
+/**
+ * SampleRequestBuilder builds sample requests that pass all request validation and get actually scored.
+ * The resulting score is likely to be useless. The purpose is to give the caller an example request to further play
+ * with and fill with actual meaningful data.
+ */
+public class SampleRequestBuilder {
+    /**
+     * Builds a valid {@link ScoreRequest} based on the given mojo input {@link MojoFrameMeta}.
+     */
+    public ScoreRequest build(MojoFrameMeta inputMeta) {
```

Comment:
can be static? if so can the class be static? if so can we have a private constructor?

Reply:
Could be, but I quite like how I can test this separately and mock it in the `RequestCheckerTest`. This would not be possible with a static method & class. Wdyt?

Reply:
There were times when I would insist... :)